src/components/Header/Header.jsx

```diff
@@ -0,0 +1,83 @@
+import "./header.css";
+import { Link } from "react-router-dom";
+import { AiOutlineSetting } from "react-icons/ai";
+import { BsSunFill } from "react-icons/bs";
+import { GiToggles } from "react-icons/gi";
+import { GrFormClose } from "react-icons/gr";
+import { useState } from "react";
+
+const Header = () => {
+
+    const [ popUpFilterAndSortDisplay, setPopUpFilterAndSortDisplay ] = useState("none");
+
+    const getThePopUpFilterAndSortDisplay =() => {
+        popUpFilterAndSortDisplay === "none" ? setPopUpFilterAndSortDisplay("flex") : setPopUpFilterAndSortDisplay("none")
+    }
+
+    const closeThePopUpFilterAndSort = () => {
+        setPopUpFilterAndSortDisplay("none")
+    }
+
+    return (
+        <div className="tn_header-component">
+            <div className="tn_header-content">
+                <Link to="/" className="tn_header-heading"><h4 className="tn_header-heading">TakeNotes</h4></Link>
+                <input type="text" placeholder="Search" className="tn_header-searchBar b-rad1" />
+                <div className="tn_icons-section">
+                    <GiToggles className="tn_header-settings-icon header-icon" onClick={() => getThePopUpFilterAndSortDisplay()} />
+                    <AiOutlineSetting className="tn_header-settings-icon header-icon" />
+                    <BsSunFill className="tn_header-lightmode-icon header-icon" />
+                    <Link to="profile">
+                        <img className="tn_header-profile-icon et_circular-avatar xs-avatar b-rad4" src="https://cdn.pixabay.com/photo/2015/12/03/08/50/paper-1074131__340.jpg" alt="profile" />
+                    </Link>
```

Comment:
`to="/profile`

Reply:
yeah!! thanks for pointing that out, didn't notice that mistake!!